src/components/safe-messages/MsgModal/index.test.tsx

```diff
@@ -305,7 +306,7 @@ describe('MsgModal', () => {
       Promise.resolve()
     })
 
-    const confirmationSpy = jest.spyOn(sender, 'dispatchSafeMsgConfirmation')
+    const confirmationSpy = jest.spyOn(sender, 'dispatchSafeMsgProposal')
```

Comment:
This changes what the test wants to test.
The test wants to test the case, that the message already exists and we just add a signature to the existing message.

So instead of changing the expected dispatch function we should mock usePollOffchainMessage to return something similar to how you changed the later test cases.